### ISSUE TITLE
Suggestions on organisation 

### ISSUE THREAD

**Issue by TheFlyingEagle** [xcp-ng/xenadmin#48]
Hi there,

I have a couple of suggestions for organizing this repository.

* Every changes to the code base through Pull Requests. This should be done to keep track of the changes that are made and gives the ability for a proper review process. 
* Evaluate the implementation for CI tests e.g. [AppVeyor](https://www.appveyor.com/)
* Proper tagging with labels, while there are already some labels (and they are good) you might want to have some differentiation between issues, e.g. `rendering` for rendering related issues and `building` for issues related to building/packing. You may also want to introduce labels like `feedback` to let all know that you are waiting for feedback
* Add the MSI installer from @gangsterrapper22 to this or a second repository, but under the "xcp-ng" flag. Some users might be worried why the installer comes from a third party repository. This keeps it clean and everything related to XenAdmin and/or xcp-ng comes from one place. 
* Close milestones that are done e.g. Release 7.4.1. Also create new milestones to show the users there is a road map on which is actively worked
* Renaming this repository to the corresponding project name `XCP-ng Center`

I also read #33 and that you guys created this PR in the upstream repository. unfortunately the PR got not merged. Why you are relying on the upstream repository? Maybe you guys should made a cut and went the road to a "full fork", meaning: Implement the changes that you thing needs to be done here and not in the upstream project. This is also a consequence of a fork - they differ and this is totally okay and fine. 

Hopefully is my tone not to harsh as it might sound, was not my intention to be rude here. Just wanted to share my thoughts here and maybe some of them are a kick off. 

Yours faithfully,
Garcia

**Comment by borzel** [comment]
We are on the road :-) it's a process and there is a lot of room for improvement. Keep in mind that we are doing this in our freetime, so the performance may not be as great as possible ;-)

Close milestones  <- god hint, I forgot that in my excitement about our first public release ;-)

**Comment by TheFlyingEagle** [comment]
Most of the points (actually all) that I listed have much time to work on, so no need to rush here. I know this repository is new and the whole things needs time, especially when you are spending your spare time on this. I just wanted to share my thoughts here and what (I think) needs to be addressed sooner or later. I saw that you already adopted a few things. :smile: :+1: 

Another point that just came to my mind:

* Create signed git tags for the releases, this builds up trust. Have a look [here](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work) for a tutorial.

Keep on going guys. :+1: 

Cheers
